### PR TITLE
endpoint 1 fini

### PR DESCRIPTION
**Pull request by RchlLou**
endpoint 1 fini

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+describe("TOPIC ENDPOINTS", () => {
+  describe("Responds with an array of all topic objects, each of with keys of 'slug' and `description`", () => {
+    it("STATUS 200: responds with an object with the key 'topics' AND value with array of topic objects", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(body).toHaveProperty("topics");
+          expect(Array.isArray(topics)).toBe(true);
```
Comment (scar1377) [inline __tests__/app.test.js:19]: No need to expect body have property "topics" as you've restructured it.

File: controllers/topics-controller.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const { retrieveTopics } = require("../models/topics-model");
+
+exports.getTopics = (req, res, next) => {
+  console.log("controller");
```
Comment (scar1377) [inline controllers/topics-controller.js]: remove console.log

**Review by scar1377 — COMMENTED** [review]
Remember to update ReadMe file

File: app.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+
+const { getTopics } = require("./controllers/topics-controller");
+
+app.get("/api/topics", getTopics);
+
+app.all("/*", (req, res) => {
+  res.status(404).send({ msg: "Endpoint not found" });
+});
```
Comment (Angaca) [inline app.js:10]: Consider to refactor errors in a separate file

**Review by Angaca — APPROVED** [review]
Good to go!

**Pull request merged**